prdoc/pr_4522.prdoc

```diff
@@ -0,0 +1,34 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Added runtime support for reporting BEEFY fork voting
+
+doc:
+  - audience:
+    - Runtime Dev
+    - Runtime User
```

Comment:
Is this description relevant to runtime users?

Reply:
I'm thinking it might be helpful for them to know of the new extrinsics.